unleash/test/no/nav/tilleggsstonader/libs/unleash/FeatureToggleServiceTester.kt

```diff
@@ -0,0 +1,18 @@
+package no.nav.tilleggsstonader.libs.unleash
+
+private val unleashService: UnleashService =
+    DefaultUnleashService(
+        apiUrl = "https://tilleggsstonader-unleash-api.nav.cloud.nais.io",
+        apiToken = System.getenv("UNLEASH_API_TOKEN"),
+        appName = "tilleggsstonader-sak",
+        strategies = emptyList(),
+    )
+
+fun main() {
+    println(unleashService.getVariant(Toggle.ROUTING_TILSYN_BARN))
+    println(unleashService.isEnabled(Toggle.ROUTING_TILSYN_BARN))
+}
```

Comment:
Vil du asserte noe her? 🤔

Reply:
Det var mer en klasse for å kunne kjøre å teste variant, mest for moro :) Men tenkte den fila kunde være nyttig å bli der